pyexchange/leverjfutures.py

```diff
@@ -239,6 +244,12 @@ def get_tickSize(self, pair: str):
         assert(isinstance(pair, str))
         return self.get_product(pair)["tickSize"]
 
+    def get_minimum_order_quantity(self, pair: str):
```

Comment:
could you add a comment in this function describing what it does?

Reply:
yup adding it